cmd/scale.go

```diff
@@ -78,6 +79,7 @@ func newScaleCmd() *cobra.Command {
 	f.BoolVar(&sc.classicMode, "classic-mode", false, "enable classic parameters and outputs")
 	f.StringVar(&sc.agentPoolToScale, "node-pool", "", "node pool to scale")
 	f.StringVar(&sc.masterFQDN, "master-FQDN", "", "FQDN for the master load balancer, Needed to scale down Kubernetes agent pools")
+	f.BoolVar(&sc.windowsPool, "windows", false, "true if windows pool")
```

Comment:
Is it necessary to have a windows flag? Would it be possible to determine if the agentpool is windows based on the names of the VMs?

Reply:
If there is no other way or it is complicated, it's fine for now but we should add a section in docs about scaling windows agent pools and mention the flag there.

Reply:
Will work on this now as it should not be a problem removing the flag, using instead the OsType of the OsDisk object in the StorageProfile (should be better than using rather the name which could change in time). In a first moment I added the flag because I thought that could have been useful, but actually it is not